src/Modules/Backend/Domain/User/Command/DisableTwoFactorAuthentication.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+namespace ForkCMS\Modules\Backend\Domain\User\Command;
+
+use ForkCMS\Modules\Backend\Domain\User\User;
+use ForkCMS\Modules\Backend\Domain\User\UserDataTransferObject;
+
+class DisableTwoFactorAuthentication extends UserDataTransferObject
```

Comment:
you don't need to extend the UserDataTransferObject here I think since all we need is the entity, just use a public readonly instead

Reply:
fixed d533e75a6455d4bcc45374065603ef440348529e